features/checkout/paying_for_order/preventing_not_available_payment_method_selection.feature

```diff
@@ -33,3 +33,14 @@ Feature: Preventing not available payment method selection
         And I complete the shipping step
         Then I should not be able to select "Cash on delivery" payment method
         And I should be able to select "Paypal Express Checkout" payment method
+
+    @api
+    Scenario: Preventing customer from selecting inexistent payment method
+        Given I have product "PHP T-Shirt" in the cart
+        And I am at the checkout addressing step
+        When I specify the billing address as "Ankh Morpork", "Frost Alley", "90210", "United States" for "Jon Snow"
+        And I complete the addressing step
+        And I select "Free" shipping method
+        And I complete the shipping step
+        And I try to select inexistent "Free" payment method
```

Comment:
I'm wondering, do we need this `inexistent` word here? 🤔 Reading the scenario from the background to this step should give us enough context to know it's the payment method that does not exist. But I know, it's me being vexatious :D